Dockerfile

```diff
@@ -1 +1,51 @@
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/centos/base-image:java8
+ENV PENTAHO_CE_VERSION=8.1.0.0-365
+ENV Mysql_Connector_Version=8.0.24
+ENV Mariadb_Connector_Version=2.2.6
```

Comment:
Please write all environment variables using `CAPITAL_SNAKE_CASE`. Thanks!

Reply:
Changed to capital's

Reply:
Please make the variables ALL CAPS.